### PR TITLE
Stack inhands

### PR DESCRIPTION
**Pull request by alsoandanswer**
inhands for stack items

also tweaks the wrapping paper sprites

dreammaker_QXikYcxWBi
dreammaker_DpDA35Ua4L
dreammaker_qOYdjo3405
dreammaker_e5Xu1pCjih
dreammaker_KIBoBCstRe

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by Doxxmedearly** [comment]
Not in love with the new wrapping paper icons but holy crap those inhands look amazing.

**Comment by NonQueueingMatt** [comment]
I think the wrappings look way, way too big.

**Comment by alsoandanswer** [comment]
> 
> 
> I think the wrappings look way, way too big.

The new ones are the smaller ones

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/stacks/nanopaste.dm [hunk 1]
```diff
@@ -3,6 +3,10 @@
 	singular_name = "nanite swarm"
 	desc = "A tube of paste containing swarms of repair nanites. Very effective in repairing robotic machinery."
 	icon = 'icons/obj/stacks/nanopaste.dmi'
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/items/stacks/lefthand_nanopaste.dmi',
+		slot_r_hand_str = 'icons/mob/items/stacks/righthand_nanopaste.dmi',
+		)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/stacks/nanopaste.dm]: Why isn't this just in the medical icons? :thinking:

Comment (alsoandanswer) [reply code/game/objects/items/stacks/nanopaste.dm]: It doesn't have the /medical/ suffix

also it's used outside of medical anyway

**Pull request merged**